### PR TITLE
MINOR: Update stale `upgrade_3_6_0` header links in documentation

### PR DESCRIPTION
**Pull request by stanislavkozlovski**

### CODE REVIEW

File: 37/upgrade.html [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
-<h4><a id="upgrade_3_6_0" href="#upgrade_3_6_0">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a></h4>
+<h4><a id="upgrade_3_6_1" href="#upgrade_3_6_1">Upgrading to 3.6.1 from any version 0.8.x through 3.5.x</a></h4>
 
-    <h5><a id="upgrade_360_zk" href="#upgrade_360_zk">Upgrading ZooKeeper-based clusters</a></h5>
+    <h5><a id="upgrade_361_zk" href="#upgrade_361_zk">Upgrading ZooKeeper-based clusters</a></h5>
     <p><b>If you are upgrading from a version prior to 2.1.x, please see the note in step 5 below about the change to the schema used to store consumer offsets.
```
Comment (stanislavkozlovski) [inline 37/upgrade.html:25]: maybe we should change the below to upgrade_361_kraft too?
Or should we just copy the section and have two identical 3.6.1 and 3.6.0 sections?

I don't know why, but through this PR I found that upgrade.html was missing the 3.6.1 and 3.5.2 changes. I introduced them with 318962858aafa5daefdde7ebda3bd30f1bc9cb4c.

As to why were they missing... 🤷‍♂️
I followed the instructions in the Wiki and introduced them on January 9 - https://github.com/apache/kafka-site/pull/576
Maybe I had an old version of kafka-site checked out locally?

File: blog.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h2 class="bullet">
                 </h2>
                 10 Oct 2023 - Satish Duggana (<a href="https://twitter.com/0xeed">@SatishDuggana</a>)
                 <p>We are proud to announce the release of Apache Kafka 3.6.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.6.0/RELEASE_NOTES.html">release notes</a>.</p>
-                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_0">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_1">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
```
Comment (stanislavkozlovski) [inline blog.html]: this is the fix

File: 37/upgrade.html (hunk 1 above)
Comment (mimaison) [reply 37/upgrade.html:25]: > Or should we just copy the section and have two identical 3.6.1 and 3.6.0 sections?

No we only keep the latest version like in https://kafka.apache.org/documentation/#upgrade

File: blog.html (hunk 2 above)
Comment (mimaison) [reply blog.html]: Let's update the title as well `Upgrading to 3.6.0` -> `Upgrading to 3.6.1`

Comment (stanislavkozlovski) [reply blog.html]: Done

**Comment by stanislavkozlovski** [comment]
This will be done with https://github.com/apache/kafka-site/pull/586/files

**Pull request closed without merging**